lib/views/subscription_slack_view.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Aprb.Views.SubscriptionSlackView do
+  import ViewHelper
+  def render(event, current_summary) do
+    %{
+      text: "",
+      attachments: "[{
+                      \"title\": \":moneybag: #{event["properties"]["partner"]["name"]}'s subscription #{event["verb"]}\",
+                      \"title_link\": \"#{admin_partners_link("subscriptions/#{event["object"]["id"]}")}\",
```

Comment:
I think it would be preferable for the helper to render a specific link (so maybe `admin_partners_subscription_link` or just `admin_subscription_link` in this case) rather than arbitrary paths. The current approach just splits URL logic between the helper and the calling code. (If there ends up being a lot of repetition of common URL parts, we can always refactor that with more helpers.)